bin/validate_json.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
I wonder if this file should live in `.github/bin/` instead? That's up to @ErikSchierboom though :D

Reply:
Basically the criterium for that is if the script is expected to also be used outside the GitHub flow. In this case, I think we won't, so `.github/bin` make sense to me.